### ISSUE TITLE
JsonWriteLongchar not working

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#1155]
Version 2.30.1

The JsonWriteLongchar rules doesn't work for me, neither in the server analysis nor in sonarLint.

```
CLASS FpTest:
   METHOD PUBLIC VOID TestMethod(
   ):
      DEFINE VARIABLE loTestObject AS Progress.Json.ObjectModel.JsonObject NO-UNDO.
      DEFINE VARIABLE llcFoo AS LONGCHAR NO-UNDO.

      loTestObject:Write(llcFoo, TRUE).
      loTestObject:Write(OUTPUT llcFoo, TRUE).
   END METHOD.
END CLASS.
```

Both writes should get flagged here.

**Comment by gquerret** [comment]
Confirmed. But note that this rule might be removed in the future as the ParameterModeInMethodCall already reports wrong method usage (with changes in #1154 )

**Comment by movedoa** [comment]
As long as that still flags OUTPUT for LONGCHAR thats fine for me
